Task_4.1/Server.cs

```diff
@@ -0,0 +1,91 @@
+using System.Runtime.CompilerServices;
+
+namespace Task_4._1;
+
+using System.IO;
+
+public class Server
+{
+    public Server(int address, int portsCount, string rootDirectory) //(int address, string rootDirectory)
+    {
+        Address = address;
+        _rootDirectory = rootDirectory;
+        _ports = new bool[portsCount];
+    }
+
+    public readonly int Address;
```

Comment:
```suggestion
    public readonly System.Net.IPAddress Address;
```
Адрес должен быть адресом, в не просто числом.